README.md

```diff
@@ -61,57 +49,70 @@ This is a basic example on how to use the library. Based on a query the language
 ```python
 # imports
 import torch
-from transformers import GPT2Tokenizer
-from trl.gpt2 import GPT2HeadWithValueModel, respond_to_batch
-from trl.ppo import PPOTrainer
+from transformers import AutoTokenizer
+from trl import PPOTrainer, PPOConfig, AutoModelForCausalLMWithValueHead
+from trl.core import respond_to_batch
 
 # get models
-gpt2_model = GPT2HeadWithValueModel.from_pretrained('gpt2')
-gpt2_model_ref = GPT2HeadWithValueModel.from_pretrained('gpt2')
-gpt2_tokenizer = GPT2Tokenizer.from_pretrained('gpt2')
+gpt2_model = AutoModelForCausalLMWithValueHead.from_pretrained('gpt2')
+gpt2_model_ref = AutoModelForCausalLMWithValueHead.from_pretrained('gpt2')
+gpt2_tokenizer = AutoTokenizer.from_pretrained('gpt2')
 
 # initialize trainer
-ppo_config = {'batch_size': 1, 'forward_batch_size': 1}
-ppo_trainer = PPOTrainer(gpt2_model, gpt2_model_ref, gpt2_tokenizer, **ppo_config)
+ppo_config = PPOConfig(
+    batch_size=1,
+    forward_batch_size=1
+)
 
 # encode a query
 query_txt = "This morning I went to the "
 query_tensor = gpt2_tokenizer.encode(query_txt, return_tensors="pt")
 
 # get model response
-response_tensor  = respond_to_batch(gpt2_model, query_tensor)
+response_tensor  = respond_to_batch(gpt2_model_ref, query_tensor)
```

Comment:
Why did you change this? I think the response should be given by the active model, no?

Reply:
Yes you are right, in this case the model is just used to create the data for the dummy dataset, but I will update it because in a normal training setting it would not be like this.